.env

```diff
@@ -50,11 +51,4 @@ HASH_TELEMETRY_DESTINATION=REPLACE_ME.aws.com
 # Is used for differentiating different apps, can be any value
 HASH_TELEMETRY_APP_ID=hash-workspace-app
 
-###########################################
-## Aliases for dockerized external services
-###########################################
-##
-## These variables are consumed in packages/hash/external-services/docker-compose.yml → env_file.
-## Using `env` → POSTGRES_USER=${HASH_PG_USER} does not give access to values in this file.
-POSTGRES_USER=${HASH_PG_USER}
-POSTGRES_PASSWORD=${HASH_PG_PASSWORD}
+ORY_KATROS_PUBLIC_URL=http://127.0.0.1:4433
```

Comment:
Kratos rather than kartos

Reply:
Good spot, thanks for catching. Addressed in https://github.com/hashintel/hash/pull/889/commits/0d309de72bd8cecc79a5b516f02630f4c7d4a33b